roles/ceph-validate/tasks/check_nfs.yml

```
@@ -6,3 +6,10 @@
     - nfs_obj_gw
     - groups.get(mon_group_name, []) | length == 0
     - (ceph_nfs_rgw_access_key is undefined or ceph_nfs_rgw_secret_key is undefined)
+
+- name: fail on openSUSE Leap 15.x using distro packages
+  fail:
+    msg: "ceph-nfs packages are not available from openSUSE Leap 15.x repositories (ceph_origin = 'distro')"
+  when:
+    - ceph_origin == 'distro'
+    - ansible_distribution == 'openSUSE Leap'
```

I think we are missing 
```suggestion
    - inventory_hostname in groups.get(nfs_group_name, [])
```

---

otherwise, all `openSUSE Leap` deployment with `ceph_origin: distro` will fail.
I guess you only want this happening when an nfs node is defined, right?

---

Those tasks are only included on nfs-related hosts, see [this part in main.yml](https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-validate/tasks/main.yml#L105). So I guess we are safe.

---

correct